<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Project_gatherClass</class>
 <widget class="QMainWindow" name="Project_gatherClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project_gather</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_login">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_startdetection">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始检测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConServer">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>连接服务器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDiscon">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>断开服务器</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>130</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSend">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_wirednetwork">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>有线正常</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_wirelessnetwork">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>无线正常</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_SMS">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>短消息正常</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>运控中心</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>连接地控中心</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>GPS/BD状态</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>正常</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>异常</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>电源系统</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>270</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>正常</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_7">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>270</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>异常</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_quit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>天线系统</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_8">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>有线正常</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_9">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>90</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>串口正常</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_commandtest">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>命令测试</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Project_gather.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_startdetection</sender>
   <signal>clicked()</signal>
   <receiver>Project_gatherClass</receiver>
   <slot>btn_startdetection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_login</sender>
   <signal>clicked()</signal>
   <receiver>Project_gatherClass</receiver>
   <slot>btn_login()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>453</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_quit</sender>
   <signal>clicked()</signal>
   <receiver>Project_gatherClass</receiver>
   <slot>btn_quit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_commandtest</sender>
   <signal>clicked()</signal>
   <receiver>Project_gatherClass</receiver>
   <slot>btn_first_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>14</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btn_login()</slot>
  <slot>btn_quit()</slot>
  <slot>btn_commandtest()</slot>
  <slot>btn_first_clicked()</slot>
 </slots>
</ui>
